var/spack/repos/builtin/packages/amg2013/package.py

```diff
@@ -33,11 +33,13 @@ class Amg2013(MakefilePackage):
     in the Center for Applied Scientific Computing (CASC) at LLNL.
     """
     tags = ['proxy-app']
-    homepage = "https://codesign.llnl.gov/amg2013.php"
-    url      = "https://codesign.llnl.gov/amg2013/amg2013.tgz"
+    homepage = "https://computation.llnl.gov/projects/co-design/amg2013"
+    url      = "https://computation.llnl.gov/projects/co-design/download/" \
+               "amg2013.tgz"
```

Comment:
No need to break URLs across multiple lines, they are automatically exempted from our flake8 checks.